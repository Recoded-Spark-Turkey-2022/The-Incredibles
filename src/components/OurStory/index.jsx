import React from 'react';
import AboutPHero from '../../assets/pics/aboutpage/support.png';
import { useTranslation } from 'react-i18next';
import Container from '../UI/Container';
function Story() {
  const [t] = useTranslation();
  return (
    <section name="story" className="">
      <div
        name="hero-image"
        className="bg-emerald-50 max-w-2xl m-auto flex justify-center items-center pt-5  "
      >
        <img src={AboutPHero} className="" />
      </div>
      <Container>
        <div
          name="story"
          className="  flex flex-col text-center items-center justify-center"
        >
          <h2 className="text-center  mb-10  sm:text-7xl   text-5xl font-bold text-cyan-700">
            {t('about.ourstory.h2')}
          </h2>

          <p className=" pb-20 sm:text-2xl text-base md:px-10">
            {t('about.ourstory.p')}
          </p>
        </div>
      </Container>
    </section>
  );
}

export default Story;
