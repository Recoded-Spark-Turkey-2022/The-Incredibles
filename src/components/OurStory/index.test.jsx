import renderer from 'react-test-renderer';
import OurStory from '.';

describe('OurStory', () =>
  it('renders correctly', () => {
    const domTree = renderer.create(<OurStory />).toJSON();
    expect(domTree).toMatchSnapshot();
  }));
