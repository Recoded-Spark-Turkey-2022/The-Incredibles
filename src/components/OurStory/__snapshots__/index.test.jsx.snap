// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OurStory renders correctly 1`] = `
<section
  className=""
  name="story"
>
  <div
    className="bg-emerald-50 max-w-2xl m-auto flex justify-center items-center  "
    name="hero-image"
  >
    <img
      className="pt-20 "
      src="support.png"
    />
  </div>
  <div
    className="container max-w-screen-xl m-auto p-10 max-md:px-5"
  >
    <div
      className="  flex flex-col text-center items-center justify-center"
      name="story"
    >
      <h2
        className="text-center  mb-10  sm:text-7xl   text-5xl font-bold text-cyan-700"
      >
        about.ourstory.h2
      </h2>
      <p
        className=" pb-20 sm:text-2xl text-base md:px-10"
      >
        about.ourstory.p
      </p>
    </div>
  </div>
</section>
`;
